.github/workflows/cli-release.yaml

```diff
@@ -70,41 +89,41 @@ jobs:
           name: cli-builds-linux-arm
           path: aptos-cli-*.zip
 
-#  build-macos-x86_64-binary:
-#    name: "Build MacOS x86_64 binary"
-#    runs-on: macos-13
-#    steps:
-#      - uses: actions/checkout@v4
-#        with:
-#          ref: ${{ github.event.inputs.source_git_ref_override }}
-#      - uses: aptos-labs/aptos-core/.github/actions/rust-setup@main
-#        with:
-#          macos: true
-#      - name: Build CLI
-#        run: scripts/cli/build_cli_release.sh "macOS" "${{inputs.release_version}}"
-#      - name: Upload Binary
-#        uses: actions/upload-artifact@v4
-#        with:
-#          name: cli-builds-macos-x86-64
-#          path: aptos-cli-*.zip
+  # build-macos-x86_64-binary:
```

Comment:
Can we delete if not used

Reply:
Still trying to get it to work, it's really a pain to remember how to do it later, so i'm leaving it commented